java/src/main/java/com/marginallyclever/filters/Filter_GeneratorYourMessageHere.java

```diff
@@ -64,10 +67,9 @@ protected void createMessage(String str,String dest) {
 			textSetAlign(Align.RIGHT);
 			textSetVAlign(VAlign.TOP);
 			textSetPosition(image_width,image_height);
-			textCreateMessageNow("Makelangelo #"+Long.toString(machine.getUID()),output);
-			
-			output.close();
```

Comment:
Sorry about removing this. Not sure why this revision made it into this branch.

I do have a branch which uses [try with resources](https://docs.oracle.com/javase/tutorial/essential/exceptions/tryResourceClose.html) making the explicit close of streams superfluous but as this is not that branch, I do not know why this was removed.